libexec/rtld-elf/rtld_c18n.c

```diff
@@ -1733,6 +1733,13 @@ _rtld_thread_start(struct pthread *curthread)
 	thr_thread_start(curthread);
 }
 
+static bool
+identify_untrusted_stk(void *canonical, void *untrusted)
+{
+	canonical = cheri_clearperm(canonical, CHERI_PERM_SW_VMEM);
```

Comment:
This change is not mentioned in the commit log.  Is it worth calling out, or perhaps splitting into a second commit?